docs/performance.robustness.rst

```diff
@@ -56,6 +56,9 @@ These show some general trends, some of which include:
 Named entity recognition
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+DaCy
+““““““““““““““““““““““““““““““
+
```

Comment:
```suggestion
```
I would make it *Named Entity Recognition* and *Fine-grained Named Entity Recognition*